toolkit/imageconfigs/packagelists/coal-packages.json

```diff
@@ -1,26 +1,26 @@
 {
     "packages": [
```

Comment:
re-review packagelist and ensure all of these new packages are absolutelty needed. Also, add a section in PR that specifies which packages are being added and why